django/db/models/base.py

```diff
@@ -569,6 +570,15 @@ def _get_pk_val(self, meta=None):
         return getattr(self, meta.pk.attname)
 
     def _set_pk_val(self, value):
+        for parent_link, (parent, field) in zip(
+                self._meta.parents.values(),
+                self._meta.parents.items()
+                ):
```

Comment:
`parent_link` and `field` is the same field, you can use `self._meta.parents.values()` without `zip` and `.items()`, e.g.
```python
for parent_link in self._meta.parents.values():
   ...
```

Reply:
Yes, my bad, I got a bit confused there. Thank you!